### DISCUSSION TITLE
Range filtering

### DISCUSSION THREAD

**Discussion by Sree-Vandana** [TanStack/table#3394] [Q&A]
__Does react-table support `range filtering on a single column` ?__
For example, I want to add two input fields for Min and Max values and after clicking search button I want the table to display values that fall in-between `[Min, Max]`  and remaining columns can have default text filter.
I cannot find any working examples showing this feature.

**Comment by KamQuoss** [comment]
Hi!
Here is working example for various types of filtering, including range filter.
https://codesandbox.io/s/condescending-rhodes-i7pr7?file=/src/App.js

According to this build in filter types:
https://github.com/tannerlinsley/react-table/blob/master/src/filterTypes.js
You should use filter: 'between'.
